### DISCUSSION TITLE
Cleaning S3 object storage

### DISCUSSION THREAD

**Discussion by Shizanit** [mastodon/mastodon#17146] [Q&A]
My instance uses backblaze B2 storage (it has S3 compatibility) for cache storage. But I ran into a big cache accumulation, how can I clear it? The usual cleanup command (screenshot below) doesn't help, although B2 is directly connected and works with S3 API without problems.
![Screenshot_20211219-223025898](https://user-images.githubusercontent.com/65107731/146689850-c1f15f6d-b715-4232-88a4-33973636265f.jpg)

**Comment by Gargron** [comment]
That command *is* supposed to work with S3 storage. In what way does it not help in your case?

**Reply by Shizanit** [reply]
For some reason, instead of deleting files becomes more by the number of how many are deleted in the console

**Reply by Shizanit** [reply]
That is, there are 5,000 files in the console remotely, and 5,000 more files in the storage

**Reply by Shizanit** [reply]
can I manually delete the external media cache folder?
